### PR TITLE
Create merged ARTIC environment dockerfile and update readme

### PR DESCRIPTION
**Pull request by AbigailShockey**

### CODE REVIEW

File: artic-ncov2019/2.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL container.version="1"
```
Comment (kapsakcj) [inline artic-ncov2019/2.0.0/Dockerfile:5]: could you please change this to `LABEL dockerfile.version="1"` ?

File: artic-ncov2019/2.0.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+LABEL website="https://github.com/artic-network/fieldbioinformatics"
+LABEL license="https://github.com/artic-network/fieldbioinformatics/blob/master/LICENSE"
+LABEL maintainer="Abigail Shockey"
+LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
+
+# install needed software for conda install
+RUN apt-get update && apt-get install -y \
+  wget \
+  git \
+  build-essential
```
Comment (kapsakcj) [inline artic-ncov2019/2.0.0/Dockerfile:18]: Could you please append this to this line? Will help keep the docker image a bit smaller
```
build-essential && \
apt-get autoclean && rm -rf /var/lib/apt/lists/*
```

File: artic-ncov2019/2.0.0/Dockerfile (hunk 1 above)
Comment (AbigailShockey) [reply artic-ncov2019/2.0.0/Dockerfile:5]: Added!

File: artic-ncov2019/2.0.0/Dockerfile (hunk 2 above)
Comment (AbigailShockey) [reply artic-ncov2019/2.0.0/Dockerfile]: Also added!

**Comment by kapsakcj** [comment]
How would you like to name this docker image and tag on dockerhub?

Looks like Kelsey pushed a docker image named `staphb/artic-ncov2019:1.1.0` 10 months ago. https://hub.docker.com/r/staphb/artic-ncov2019/tags?page=1&ordering=last_updated

Not sure if I should add this new dockerfile as a new tag there, or start a new dockerhub repository named something else

**Comment by kapsakcj** [comment]
And one more suggestion for the Dockerfile. You can save a decent amount of space if you run `conda clean -a -y` after creating the conda environment. I _think_ everything will still run OK, but would be worth a quick test

```
REPOSITORY          TAG                 IMAGE ID       CREATED          SIZE
abigail/artic       latest-condaclean   d5a31b2310e1   4 minutes ago    2.71GB
abigail/artic       latest              42d44e4d7f42   10 minutes ago   3.46GB
```
I just added the command to this step:
```Dockerfile
# create the conda environment and set as default
RUN conda env create -f /fieldbioinformatics/environment.yml &&\
  echo "source activate artic" > /etc/bash.bashrc &&\
  cd /fieldbioinformatics/ &&\
  python setup.py install &&\
  conda clean -a -y
```

**Comment by AbigailShockey** [comment]
This should be named staphb/artic-ncov2019:2.0.0

**Comment by kapsakcj** [comment]
I was able to run `artic -v` & `./test-runner.sh medaka` & `./test-runner.sh nanopolish` in the conda clean version of the image. I think it still runs everything as expected?

**Comment by AbigailShockey** [comment]
I added the clean step and tested it. Should be good to go

**Comment by kapsakcj** [comment]
Sweet! I'll merge and push to dockerhub. Thanks for making those changes!

**Pull request merged**